### PR TITLE
61455 ra oauth2 validation

### PR DESCRIPTION
**Pull request by nilsgstrabo**

### CODE REVIEW

File: pkg/apis/radixvalidators/validate_ra.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ const (
 )
 
 var (
-	validOAuthSessionStoreTypes []string = []string{"", string(radixv1.SessionStoreCookie), string(radixv1.SessionStoreRedis)}
+	validOAuthSessionStoreTypes []string = []string{string(radixv1.SessionStoreCookie), string(radixv1.SessionStoreRedis)}
 	validOAuthCookieSameSites   []string = []string{string(radixv1.SameSiteStrict), string(radixv1.SameSiteLax), string(radixv1.SameSiteNone), string(radixv1.SameSiteEmpty)}
 	illegalVariableNamePrefixes          = [...]string{"RADIX_", "RADIXOPERATOR_"}
```
Comment (satr) [inline pkg/apis/radixvalidators/validate_ra.go:39]: Side-note - this is not used: `illegalVariableNamePrefixes          = [...]string{"RADIX_", "RADIXOPERATOR_"}`

**Review by satr — CHANGES_REQUESTED** [review]
Some comments

File: pkg/apis/radixvalidators/validate_ra.go [hunk 2]
```diff
@@ -34,7 +34,7 @@ const (
 )
 
 var (
-	validOAuthSessionStoreTypes []string = []string{"", string(radixv1.SessionStoreCookie), string(radixv1.SessionStoreRedis)}
+	validOAuthSessionStoreTypes []string = []string{string(radixv1.SessionStoreCookie), string(radixv1.SessionStoreRedis)}
 	validOAuthCookieSameSites   []string = []string{string(radixv1.SameSiteStrict), string(radixv1.SameSiteLax), string(radixv1.SameSiteNone), string(radixv1.SameSiteEmpty)}
```
[the remark this answers is not in the document]
Comment (nilsgstrabo) [reply pkg/apis/radixvalidators/validate_ra.go:38]: fixed

File: pkg/apis/radixvalidators/validate_ra.go (hunk 1 above)
Comment (nilsgstrabo) [reply pkg/apis/radixvalidators/validate_ra.go:39]: removed

**Review by satr — APPROVED** [review]

**Pull request merged**